server/src/main/java/org/elasticsearch/index/search/stats/SearchStats.java

```diff
@@ -154,7 +154,10 @@ public long getFetchCount() {
         }
 
         public TimeValue getFetchTime() {
-            return new TimeValue(fetchTimeInMillis);
+            if (fetchTimeInMillis >= -1) {
```

Comment:
Wait, this shouldn't be necessary? I see tests we are passing in `-1`? Those aren't realistic though, right?

Reply:
Reverted and fixed the tests, see above.